panels/templates/panel_app_management/assigned_to.html

```diff
@@ -6,6 +6,9 @@
 
 You can download a CSV spreadsheet of all of {{ attendee.first_name }}'s panels
 <a href="panels_by_poc?poc_id={{ attendee.id }}">here</a>.
+</br>
+You can view a schedule of all of {{ attendee.first_name }}'s panels
+<a href="staff_schedule?attendee_id={{ attendee.id }}">here</a>.
```

Comment:
Another minor naming suggestion, but I was briefly confused by the fact that this says "staff" but the other link says "poc." I see from the issue that those are the same thing in this case, so it'd be easier for future readers to name this page `panel_poc_schedule` or something.